_javascript/lib/sidebarNavigation.js

```diff
@@ -13,19 +13,39 @@ export default (links) => {
   const scrollToLink = (element) => {
     const locationID = element.getAttribute("data-target");
     const targetElement = document.getElementById(locationID);
+    const y = getElementLocation(targetElement);
+
+    let intervalID;
 
     window.scrollTo({
-      top: getElementLocation(targetElement),
+      top: y,
       behavior: "smooth",
     });
 
-    if (window.innerWidth < 768) {
-      const body = document.querySelector("body");
+    intervalID = setInterval(function () {
```

Comment:
```suggestion
    const intervalID = setInterval(function () {
```

Don't need to split it and you don't reassign it.